adrenaline/snaps_adrenaline/playbooks/consts.py

```diff
@@ -77,8 +77,8 @@
 K8S_DEPOY_NODE_CONFIG_TMPLT = pkg_resources.resource_filename(
     K8S_TMPLT_PKG, 'node_config.j2')
 
-FPGA_K8S_SPEC_URL = 'https://raw.githubusercontent.com/Xilinx/FPGA_as_a_Service/master/k8s-fpga-device-plugin/trunk/fpga-device-plugin.yml'
-GPU_K8S_SPEC_URL = 'https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/v0.6.0/nvidia-device-plugin.yml'
+FPGA_K8S_SPEC_URL = 'https://raw.githubusercontent.com/Xilinx/FPGA_as_a_Service/master/k8s-fpga-device-plugin/fpga-device-plugin.yml'
+GPU_K8S_SPEC_URL = 'https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/master/nvidia-device-plugin.yml'
 
```

Comment:
Instead of using "master" which will be changing in the future, is it possible to pin to specific version that we know it's working for current version of snaps-kubernetes?

Reply:
for GPU,  the plugin is pinned to v0.7.0 
for FPGA, there are no versions, no tags, no active branches. Only master is available. hence retaining the same.

Reply:
Looks good to me.